nextflow.config

```diff
@@ -102,16 +102,17 @@ params {
     // MALT
     run_malt                   = false
     malt_mode                  = 'BlastN'
-    malt_generatemegansummary = false
+    malt_generate_megansummary = false
+    malt_save_reads            = false
 
     // kraken2
-    run_kraken2                = false
+    run_kraken2                     = false
+    kraken2_save_reads              = false
+    kraken2_save_readclassification = false
```

Comment:
Maybe I misunderstand something here, but what is the difference between `kraken2_save_reads` and `kraken2_save_readclassification`?